src/VaultLogger.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace SublimeSkinz\SublimeVault;
+
+use Analog\Logger;
+use Analog\Handler\Stderr;
+use Analog\Handler\File;
+
+class VaultLogger extends Logger
+{
+
+    public function __construct()
+    {
+        $logger = new Logger();
+
+        switch (getenv('LOGGER_HANDLER')) {
```

Comment:
Great :+1: It is possible to have something like that to manage raw log vs json log ?

Reply:
yes :)